scala/lambdas/dynamo-formatters/src/main/scala/uk/gov/nationalarchives/dynamoformatters/DynamoWriteUtils.scala

```diff
@@ -43,7 +43,6 @@ object DynamoWriteUtils {
           "transferCompleteDatetime" -> DynamoValue.fromString(assetDynamoItem.transferCompleteDatetime.toString),
           "upstreamSystem" -> DynamoValue.fromString(assetDynamoItem.upstreamSystem),
           "digitalAssetSource" -> DynamoValue.fromString(assetDynamoItem.digitalAssetSource),
-          "digitalAssetSubtype" -> DynamoValue.fromString(assetDynamoItem.digitalAssetSubtype),
           "originalFiles" -> DynamoValue.fromStrings(assetDynamoItem.originalFiles.map(_.toString)),
```

Comment:
I think this still needs to be in here. We still want to write it to Dynamo if it's set. 
Can you add it to the bottom of this method like the one for correlationId
```scala
++ assetDynamoItem.potentialDigitalAssetSubtype.map(subType => Map("digitalAssetSubtype" -> DynamoValue.fromString(subType))).getOrElse(Map())
```